databases/sql-server/schema.sql

```diff
@@ -0,0 +1,35 @@
+CREATE DATABASE prisma;
```

Comment:
I would suggest to use a different name that sounds more like a "project name", a few examples:

- `prisma-demo`
- `my-app`
- `hello-prisma`